config/routes.rb

```diff
@@ -2,5 +2,6 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   scope module: 'api/v1', path: 'api/v1' do
     post '/games', to: 'games#create', as: :create_game
+    post '/games/:invite_code/players', to: 'games#join', as: :join_game
```

Comment:
Is this for creating players or for starting the game?

Reply:
This will be for invited players to join an existing game, before the game begins.